src/platform/forms-system/src/js/helpers.js

```diff
@@ -701,3 +701,24 @@ export function transformForSubmit(
 
   return JSON.stringify(withoutViewFields, replacer) || '{}';
 }
+
+/**
+ * This function accepts a uiSchema object and returns its closest child object that contains the uiSchema entries
+ *
+ * Note: This is not intended for uiSchema child properties that have more than one sibling. Support for this is still a work in progress.
```

Comment:
How will this impact forms that have uiSchema child properties with more than one sibling?